deps/exokit-bindings/canvascontext/include/canvas-context.h

```diff
@@ -31,18 +33,19 @@ class ImageBitmap;
 class Path2D;
 class CanvasGradient;
 
-enum class TextBaseline {
-  TOP,
-  HANGING,
-  MIDDLE,
-  ALPHABETIC,
-  IDEOGRAPHIC,
-  BOTTOM,
-};
-enum class Direction {
-  LEFT_TO_RIGHT,
-  RIGHT_TO_LEFT,
-};
+static const char* kTextBaseline_TOP = "top";
```

Comment:
Why are text constants used here, instead of an enum? There seems to be pointer comparisons against these, which is fine, but things could get confusing later when textBaseline comes from user input -- in which case such comparisons would not do the right thing.

If we need to print these, perhaps an enum + stringifier is the more foolproof option.

Reply:
It avoids large if-this-then-string chains in every getter.

textBaseline already comes from user input, doesn't it? That was partly why I wanted to convert everything to strings. Then I got rid of the string comparisons and ended up with that.


The downside is a lack of type safety. And I dislike that if anyone makes a mistake, the result is a crash instead of an invalid value. I'll change it back later.